File: OCH_Tao/ABC/B/093.js
const [A, B, K] = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
let l = [];
for (let i = A; i < A + K; i++) {
  l.push(i);
}
for (let i = B; i > B - K; i--) {
  l.push(i);
}
l = [...new Set(l)];
l = l.filter(x => A <= x && x <= B);
l.sort((a, b) => a - b);
console.log(l.join("\n"));

File: OCH_Tao/ABC/B/093.py
A,B,K = map(int,input().split())
l = []
l.extend(list(range(A,A+K)))
l.extend(list(range(B,B-K,-1)))
l = list(set(l))
l = [i for i in l if A<=i<=B]
l.sort()
print("\n".join([str(i) for i in l]))